src/main/webapp/provided.js

```diff
@@ -0,0 +1,60 @@
+// provided.js can be overriden in order to have a predefined-report generated.
```

Comment:
Why do you added the provided.js ?

Reply:
There is no reason.
I simply checked out [JMH-Vizualizer](https://github.com/jzillmann/jmh-visualizer) and build it.
After that I moved all build-artifact's into the associated resources directory.

After that the plugin-ui has some new features which are working and looking nice.
Hence I thought everything is fine.

If some of the resources are unnecessary you could remove it or adopt it (like the settings.js).